packages/babel-plugin-polyfill-corejs3/src/shipped-proposals.ts

```diff
@@ -1,8 +1,10 @@
 // This file is automatically generated by scripts/build-corejs3-shipped-proposals.mjs
 
 export default new Set<string>([
-  "esnext.array.find-last",
-  "esnext.array.find-last-index",
-  "esnext.typed-array.find-last",
-  "esnext.typed-array.find-last-index",
+  "esnext.array.to-reversed",
```

Comment:
Is it already available in some browsers by default?

Reply:
It's generated automatically, so yes it's available in some *environments* by default. It's not a browser, it's `bun`.

Reply:
But `preset-env`'s shippedProposals does not include `bun`. Until `preset-env` recognizes `bun` as shippedProposals, can we exclude `bun` here?

https://github.com/babel/babel-polyfills/blob/ce30946033aaed0ae1daf09a75e76acf4d24b8cc/scripts/build-corejs3-shipped-proposals.mjs#L21

Reply:
I think that it should be solved on the `preset-env` side, https://github.com/babel/babel/issues/14831.

Reply:
So, this proposal is supported in Safari 16 that's released today. If you wanna change something in the `shippedProposals` logic - feel free to do it out of the scope of this PR - I think that now it's incorrect anyway.